src/models/Request.js

```diff
@@ -46,6 +46,11 @@ export default (sequelize, DataTypes) => {
       type: DataTypes.ENUM,
       values: ['BUSINESS', 'VACATION', 'EXPEDITION'],
       defaultValue: 'BUSINESS'
+    },
+    status: {
+      type: DataTypes.ENUM,
+      values: ['pending', 'approved', 'rejected'],
+      defaultValue: 'pending'
     }
```

Comment:
Whys is `status` defined as an `ENUM` in the model definition and it is defined as a `STRING` in the migration file?